dev-java/failureaccess/failureaccess-32.1.3.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P=guava-${PV}
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="com.google.guava:failureaccess:1.0.1"
```

Comment:
is the version correct?

Reply:
Yes, it is, see https://github.com/google/guava/blob/v32.1.3/futures/failureaccess/pom.xml#L11
They keep `1.0.1` for several guava versions. I am doing this brainless bump in order to avoid another guava tarball.